sklearn/_build_utils/tempita.py

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env python3
+
```

Comment:
For the reason why this shebang has been added, please refer to https://github.com/numpy/numpy/pull/24971.